scripts/scil_smooth_surface.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to smooth surface with a Laplacian blur.
+
+step_size in between 0.1 to 10 are recommended
```

Comment:
step_size _in_ between 0.1 to 10 **is** recommended
(Not sure about the in between, your call)